django_toolbox/discounts/draft_order.py

```diff
@@ -88,6 +88,7 @@ def create_draft_order(shop: AuthAppShopUser, data, get_offers_line_items):
             "appliedDiscount": discount.apply_all_items_discount(),
             "note": cart["note"],
             "shippingAddress": data.get("shippingAddress"),
+            "metafields": tracking_data,
```

Comment:
Ja myslel, ze ty metafields budou definovane tady, proto jsem rikal `tracking_data`. Tohle reseni se mi ale taky libi, jen to pojmenuj `metafields` :)